File: tuan01/tong.py
import sys

input_data = sys.stdin.read().split()

def main():
    if not input_data:
        return
        
    n = int(input_data[0])
    a = [int(x) for x in input_data[1:n+1]]
    
    sumv = sum(a) # dung luon ham sum va max co san
    maxv = max(a)
    
    print(f"{sumv} {maxv}")

if __name__ == "__main__":
    main()

File: tuan01/tong.jl
function main()
    input_text = read(stdin, String)
    tokens = split(input_text)
    
    if isempty(tokens)
        return
    end
    
    n = parse(Int64, tokens[1])
    a = [parse(Int64, tokens[i]) for i in 2:(n+1)]
    
    sumv = sum(a) # tinh tong va tim max bang ham cua julia
    maxv = maximum(a)
    
    println("$sumv $maxv")
end

main()

